README.md

```diff
@@ -29,22 +29,35 @@ OpenTelemetry JS is under active development.
 This release isn't guaranteed to conform to a specific version of the specification, and future
 releases will not attempt to maintain backwards compatibility with the alpha release.
 
-| Component                   | Version      | Release Date     |
-| --------------------------- | ------------ | ---------------- |
-| Tracing API                 | Alpha v0.1.0 | October 14 2019  |
-| Tracing SDK (Node and Web)  | Alpha v0.1.0 | October 14 2019  |
-| Jaeger Trace Exporter       | Alpha v0.1.0 | October 14 2019  |
-| Trace Context Propagation   | Alpha v0.1.0 | October 14 2019  |
-| Zipkin Trace Exporter       | Alpha v0.1.0 | October 14 2019  |
-| OpenTracing Bridge          | Alpha v0.1.0 | October 14 2019  |
-| Metrics API                 | Alpha v0.2.0 | November 04 2019 |
-| Metrics SDK                 | Alpha v0.2.0 | November 04 2019 |
-| Prometheus Metric Exporter  | Alpha v0.3.0 | December 13 2019 |
-| Collector Exporter          | Alpha v0.3.0 | December 13 2019 |
-| Resources                   | Beta v1      | March 16 2020    |
-| Support for Tags/Baggage    | Beta v1      | March 16 2020    |
-| Metrics SDK (Complete)      | Beta v1      | March 16 2020    |
-| OpenCensus Bridge           | Beta v1      | Unknown          |
+| Component                   | Initial Version | Release Date     |
+| --------------------------- | --------------- | ---------------- |
+| Tracing API                 | Alpha v0.1.0    | October 14 2019  |
+| Tracing SDK (Node and Web)  | Alpha v0.1.0    | October 14 2019  |
+| Jaeger Trace Exporter       | Alpha v0.1.0    | October 14 2019  |
+| Trace Context Propagation   | Alpha v0.1.0    | October 14 2019  |
+| Zipkin Trace Exporter       | Alpha v0.1.0    | October 14 2019  |
+| OpenTracing Bridge          | Alpha v0.1.0    | October 14 2019  |
+| Metrics API                 | Alpha v0.2.0    | November 04 2019 |
+| Metrics SDK                 | Alpha v0.2.0    | November 04 2019 |
+| Prometheus Metric Exporter  | Alpha v0.3.0    | December 13 2019 |
+| Resources                   | Beta v0.5.0     | March 16 2020    |
+| Metrics SDK (Complete)      | Beta v0.5.0     | March 16 2020    |
+| OpenCensus Bridge           | Unknown         | Unknown          |
+| Support for Tags/Baggage    | Unknown         | Unknown          |
+
+## Feature Status
+
+Last updated March 2020
+
+| Feature             | API Status | API Target | SDK Status  |
+| ------------------- | ---------- | ---------- | ----------- |
+| Tracing             | Beta       | v0.4       | Beta        |
```

Comment:
`s/v0.4/v0.5/g` ?

Reply:
I think the API version is actually 4. i'll check

Reply:
it's actually 3 https://github.com/open-telemetry/opentelemetry-specification/blob/master/CHANGELOG.md

Reply:
Is this wording more clear? https://github.com/open-telemetry/opentelemetry-js/pull/864/commits/73f7f41e7fae021b334d8365791a982d237a74c0